### PR TITLE
Frogger js core code

### PR DESCRIPTION
**Pull request by AsaMitaka**
### Object oriented JS

[Demo](https://asamitaka.github.io/frontend-nanodegree-arcade-game)
[Code](https://github.com/AsaMitaka/frontend-nanodegree-arcade-game/blob/master/js/app.js)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

File: submissions/asaMitaka/frogger/app.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
+// Enemies our player must avoid
```
Comment (artem-trubin) [inline submissions/asaMitaka/frogger/app.js:1]: Seems like an irrelevant comment here.

File: submissions/asaMitaka/frogger/app.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+var newP = document.createElement('p');
```
Comment (artem-trubin) [inline submissions/asaMitaka/frogger/app.js:2]: You should avoid using `var`s. Most of the time to store a value `const` is enough and only if you are 100% sure that you'll change this value in code later use `let`.

File: submissions/asaMitaka/frogger/app.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+newP.textContent = `Your score: 0 `;
+document.body.append(newP);
+
+var enemyStat = {
```
Comment (artem-trubin) [inline submissions/asaMitaka/frogger/app.js:6]: There is no reason for a separate object that stores information about enemies since you already has a whole class for them. Store these values there.

File: submissions/asaMitaka/frogger/app.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+}
+
+
+Player.prototype.update = function() {
+    this.win();
+}
+
+Player.prototype.win = function() {
+    if (field.min >= this.posY) {
+        console.log(field.min, this.posY);
```
Comment (artem-trubin) [inline submissions/asaMitaka/frogger/app.js:111]: `console.log` is fine for debugging but should be removed from the final version of the code.

**Review by artem-trubin — COMMENTED** [review]
Hey! There are some improvements that can be done, but in general, nice work! 

You've chosen to make the character go not cell-by-cell but with smaller steps which are fine but it's a little strange that it can go so low, almost off the level, and stops going left so early. Please work on the limits of where the character can go.

Also, check comments below.

**Comment by artem-trubin** [comment]
Did you forget to update your own repo? Github Pages demo still has problems listed in my previous review.

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
Also follow the requirements here https://github.com/kottans/frontend-2022-homeworks/pull/92#issuecomment-1216478616

Feel free asking questions here or in Students' chat if anything is not really clear.

When done, please confirm explicitly that all requirements are met.

**Comment by artem-trubin** [comment]
Great job!
